src/args.rs

```diff
@@ -12,6 +12,12 @@ fn parse_speed_factor(s: &str) -> Result<f32, String> {
     Ok(value_as_float)
 }
 
+#[cfg(not(target_arch = "wasm32"))]
+fn parse_instant_print(s: &str) -> Result<u32, String> {
+    let value_as_int = s.parse::<u32>().map_err(|e| e.to_string())?;
+    Ok(value_as_int)
+}
```

Comment:
Do you actually need this? If you just specify `u32`, it should already parse this correctly I think.